### ISSUE TITLE
Bug: custom sounds are removed on reboot

### ISSUE THREAD

**Issue by tmdag** [rafaelmardojai/blanket#348]
I thought it was due to the system upgrade, but unfortunately no. With each reboot, all my custom sounds are getting removed.

I'm using Blanket 0.7.0 (installed via flathub)
Kernel: 6.8.7-200.fc39.x86_64
Gnome Wayland

![Screenshot from 2024-04-24 11-16-32](https://github.com/rafaelmardojai/blanket/assets/9139248/120ce50e-723e-44c1-837e-d0fcad02e0ca)

**Comment by tmdag** [comment]
Same issue on Fedora 39. I don't remember having this issue on fedora 38. I would assume its has something to do with flatpack itself maybe

**Comment by rafaelmardojai** [comment]
Yes, I need to investigate a solution, but this is caused by the portal used in flatpak to get file access. For some reason the app will lose access to that file.

**Comment by bobby-abe** [comment]
@rafaelmardojai 
Any updates on this issue? LOVE the app but really need my custom sounds as part of  my sound mix.

**Comment by rafaelmardojai** [comment]
@bobby-abe Not yet, I need to investigate further and report the problem to the correct component in the stack. As a workaround I guess you could give Blanket filesystem permission `home`  using something like Flatseal, that should avoid the document portal.

**Comment by rafaelmardojai** [comment]
Duplicated of  #306
